website/library/reference/how-to-review-someone's-work.md

```diff
@@ -0,0 +1,47 @@
+# How to be a great peer reviewer
+
+When you first start your technical writing journey, one of the many important skills you'll need to learn is how to do is peer reviews. It's a skill that will not only teach you how to collaborate with other technical writers and other people who interact with documentation, but it will also help you to become a better technical writer in the long run.
+
+## Tips for doing effective Peer Reviews
+
+When it comes to peer reviews, it can get confusing and overwhelming at times because you want to be effective, but at the same time you don't want to give too much feedback to point that the TA is confused. Consider using this checklist of things to keep in mind when doing peer reviews:
+
+### Content Review
+
+* **Overall structure and flow**:
+  * Are the steps presented in a logical sequence that guides the user effectively?
+  * Is the language simple for the user to understand?
+  * Have all the terms been clearly defined?
+* **Accuracy and completeness**:
+  * Does the document present any information that could be unclear or misleading to the user?
+  * Are all steps accurate and free from mistakes?
+  * Do the steps' code snippets work when followed?
+
+### Clarity and Readability
+
+* **Formatting and consistency**:
+  * Are headings and subheadings used consistently?
+  * Are lists and tables used effectively?
+  * Are images and diagrams used to support the text?
+
+### Mechanics and Style
+
+* **Spelling and grammar**:
+  * Are there any spelling or grammar errors?
+  * Are punctuation and capitalization consistent?
+* **Style and tone**:
+  * Is the tone consistent throughout the document?
+  * Are there any areas where the tone could be improved?
+
+### Additional Checks
+
+* **Links and references**:
+  * Are all links working correctly?
+  * Are references properly cited?
+* **Examples**:
+  * Do the real-world examples effectively enhance the understanding of the documentation's topic?
+  * Do the real-world examples resonate with the user?
+
+## Conclusion
```

Comment:
I think, rather than having a conclusion, this should go somewhere near the beginning to explain why reviews are so important / fundamental, and why they're so beneficial both to the writer and to whatever we're documenting. Perhaps with a little explanation on how a different perspective on what's been written is so important, and can highlight assumptions as well as inconsistencies, as well as what tone you should try to adopt - supportive and encouraging, for example, while also being clear and exact. What do you think?

Reply:
That does work better now that I think about it @degville